2.6/index.html

```diff
@@ -128,6 +132,7 @@ <h2 id="thanks">Thanks</h2>
         </div>
         <footer>
             <p>Code on <a href="https://github.com/hugovk/drop-python">GitHub</a>.</p>
+            <a class="github-fork-ribbon" href="https://github.com/hugovk" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
```

Comment:
Please could you update these to link to the repo rather then my user page?

https://github.com/hugovk/drop-python/

Reply:
fixed!